src/types/SortSex.ts

```diff
@@ -0,0 +1,5 @@
+export enum SortSex {
+  all = 'all',
```

Comment:
Same here, pls capitalize the enum keys to follow common naming conventions for enums.